root/oauth2/Oob.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import Layout from '../layout';
+import {l} from '../static/scripts/common/i18n';
+
+type Props = {|
+  +application: ApplicationT,
+  +code: string,
+|};
+
+const OAuth2Oob = ({
+  application,
+  code,
+}: Props) => (
+  <Layout fullWidth title={l('OAuth Authorization')}>
+    <h1>{l('Success!')}</h1>
+    <p>
+      {l(
+        `You have granted access to {app}. Next, return to {app} and
+         copy this token to complete the authorization process:`,
+        {app: application.name},
+      )}
+    </p>
+    <p><code>{code}</code></p>
+  </Layout>
+);
+
+export default OAuth2Oob;
```

Comment:
Why not call the file OAuth2Oob.js? Same for the others, especially Oauth2Error

Reply:
Hm, I guess we're not very consistent about that, e.g. all the files named `/Index`, `/List`, `/Results`, `/NotFound` etc. that don't repeat the entity/subpath name. Not sure which I prefer. I guess it's nice if the file name matches the component name, but shorter/non-repetitive paths are nice too and it doesn't change the name you import by

Reply:
I think I would rather have unique names when possible, but maybe that's just me.

Reply:
Fixed to use the full component name in the file names.